error.go

```diff
@@ -112,25 +113,31 @@ func (e errConstructorFailed) Error() string {
 // errArgumentsFailed is returned when a function could not be run because one
 // of its dependencies failed to build for any reason.
 type errArgumentsFailed struct {
+	c      *Container
```

Comment:
We can export these fields and make them more descriptive:
```
type errArgumentsFailed struct {
  Container *Container
  ...
}
```

Reply:
Renamed and made exported!